gallery/src/components/NxAccordion/NxAccordionPage.tsx

```diff
@@ -4,184 +4,265 @@
-            the <NxCode>.nx-accordion__header-title</NxCode> class containing
-            the text content of the always-visible section of the accordion. Note
-            that <NxCode>NxAccordion</NxCode> is a{' '}
-            <NxTextLink external href="https://html.spec.whatwg.org/multipage/sections.html#sectioning-root">
-              sectioning root
-            </NxTextLink>
-            , so it does not matter whether this header is a lower-rank heading than that of its surrounding
-            section.
-          </NxP>
-        </NxTile.SubsectionHeader>
-      </NxTile.Subsection>
-      <NxTile.Subsection>
-        <NxTile.SubsectionHeader>
-          <NxH3>Helper Classes</NxH3>
-        </NxTile.SubsectionHeader>
-        <NxP>The following CSS classes are available for use on child elements.</NxP>
-        <NxTable>
-          <NxTable.Head>
-            <NxTable.Row>
-              <NxTable.Cell>Name</NxTable.Cell>
-              <NxTable.Cell>Location</NxTable.Cell>
-              <NxTable.Cell>Description</NxTable.Cell>
-            </NxTable.Row>
-          </NxTable.Head>
-          <NxTable.Body>
-            <NxTable.Row>
-              <NxTable.Cell><NxCode>nx-btn-bar</NxCode></NxTable.Cell>
-              <NxTable.Cell>Last child of <NxCode>NxAccordion.Header</NxCode></NxTable.Cell>
-              <NxTable.Cell>
-                <NxCode>NxAccordion.Header</NxCode> supports the inclusion of buttons on
-                its right-hand side. This is accomplished by adding
-                an <NxCode>.nx-btn-bar</NxCode> after
-                the <NxCode>NxAccordion.Title</NxCode>.
-              </NxTable.Cell>
-            </NxTable.Row>
-            <NxTable.Row>
-              <NxTable.Cell><NxCode>nx-h3</NxCode></NxTable.Cell>
-              <NxTable.Cell>Subheader within accordion body</NxTable.Cell>
-              <NxTable.Cell>
-                The contents of the accordion body may include subheaders which should
-                be <NxCode>&lt;h3&gt;</NxCode> elements with
-                the <NxCode>.nx-h3</NxCode> class.
-              </NxTable.Cell>
-            </NxTable.Row>
-          </NxTable.Body>
-        </NxTable>
-      </NxTile.Subsection>
-    </GalleryDescriptionTile>
+              Note that this component is stateless – its open state must be tracked externally.
+              See <NxCode>NxStatefulAccordion</NxCode> for a version which tracks its own open state.
+            </NxP>
+            <NxH3>Stateful Accordion</NxH3>
+            <NxP>
+              <NxCode>NxStatefulAccordion</NxCode> is a wrapper
+              around <NxCode>NxAccordion</NxCode> which tracks its own toggle state. It accepts
+              the same props as <NxCode>NxAccordion</NxCode>, except that instead
+              of <NxCode>open</NxCode>, it accepts <NxCode>defaultOpen</NxCode> which
+              provides the initial toggle state.
+            </NxP>
+            <NxH2>Guidelines</NxH2>
```

Comment:
I don't think it's valid either in an HTML-outline sense or in a kaleido sense to have a second h2 partway down a tile

Reply:
> I don't think it's valid either in an HTML-outline sense or in a kaleido sense to have a second h2 partway down a tile

https://github.com/sonatype/sonatype-react-shared-components/pull/607/commits/cc91e7c1fa1737f501917c78fa45f8119d43f9d7